pkg/parser/parser_test.go

```diff
@@ -189,28 +189,44 @@ x=len('123')
 
 func TestFuncDecl(t *testing.T) {
 	input := `
-c := add 1 2
-func add:num n1:num n2:num
+c := 10
+func nums1:num n1:num n2:num
 	if true //TODO: > 10
 	    print c
 	    return n1
 	end
-	return n1 // + n2
+	return n2
 end
 on mousedown
 	if true //TODO: > 10
 	    print c
 	end
 end
+func nums2:num n1:num n2:num
+	if true //TODO: > 10
+		return n1
+	else
+		return n2
+	end
+end
+func nums3:num
```

Comment:
is this function for later? because it should be invalid since it does not return a value at the end, and you do not have that constant condition checking we've talked about. Right now, I would expect this to generate an error and only later when  you add that constant checking would it pass.

Do you have any checking for missing returns on functions yet?

Reply:
yeah, I don't have checks for missing returns yet. I should probably add this to this PR in a last commit.

Reply:
yeah, I don't have checks for missing returns yet. I should probably add this to this PR in a last commit.